openapi.yml

```diff
@@ -55,9 +55,9 @@ paths:
 components:
   schemas:
     Address:
-      description: Ethereum 40 byte address encoded as a hex.
+      description: Ethereum 40 byte address encoded as a hex without `0x` prefix.
```

Comment:
I've changed the api here because the 0x seems more annoying than useful. Most hex libraries do not use so I imagine this is easier for the frontend and the backend.